### DISCUSSION TITLE
Big doc updates around the release of v16.8.0?

### DISCUSSION THREAD

**Discussion by smikitky** [reactjs/reactjs.org-translation#227] [Team Posts]
@tesseralis What are the current plans of doc updates around the release of 16.8? Just remove `(preview)` from Hooks? Or is there going to be a big rewrite which may nullify part of the current translation?

**Comment by gaearon** [comment]
This is my work in progress PR: https://github.com/reactjs/reactjs.org/pull/1593

There will be minor tweaks and additions but nothing large.

**Comment by tesseralis** [comment]
And I should have the bot set up for tonight so we don't have to worry about manually cherry-picking.
